<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridDockWidget</class>
 <widget class="QDockWidget" name="GridDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Grid</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Grids irregulares</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="cajaRadioButton">
          <property name="text">
           <string>Caja envolvente</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="hadBoton">
              <property name="minimumSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/HAD32.png</normaloff>:/iconos/images/HAD32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="haiBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/HAI32.png</normaloff>:/iconos/images/HAI32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="habdBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/HABD32.png</normaloff>:/iconos/images/HABD32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="habiBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/HABI32.png</normaloff>:/iconos/images/HABI32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="vabdBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/VABD32.png</normaloff>:/iconos/images/VABD32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="vabiBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/VABI32.png</normaloff>:/iconos/images/VABI32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="vadBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/VAD32.png</normaloff>:/iconos/images/VAD32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="vaiBoton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos.qrc">
                <normaloff>:/iconos/images/VAI32.png</normaloff>:/iconos/images/VAI32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="verCajaCheckBox">
          <property name="text">
           <string>Ver caja envolvente</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Grids básicos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="poligonoRadioButton">
          <property name="text">
           <string>Polígono</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Opciones</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="areaLabel">
            <property name="text">
             <string>Área del grid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="areaGridTextEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ha</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="visualizarRutaCheckBox">
          <property name="text">
           <string>Mostrar previa ruta</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="overshotLabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Overshot</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="overshotTextEdit">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="metros1Label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>metros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="overshotlatLabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Overshot lat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="overshotLatEditText">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="metros2Label">
            <property name="text">
             <string>metros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="camaraLabel">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Cámara</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="anguloLabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Ángulo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="anguloDial">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="anguloSpinBox">
            <property name="maximum">
             <number>359</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="gradosAnguloLabel">
            <property name="text">
             <string>grados</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="alturaLabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Altura de vuelo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="alturaVueloTextEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>120</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="metros3Label">
            <property name="text">
             <string>metros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>GSD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="gsdText"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>cm/px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="solapedelanteroLabel">
            <property name="text">
             <string>Solape delantero</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="solapeDelanteroSpinBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="porcentaje1Label">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="solapelateralLabel">
            <property name="minimumSize">
             <size>
              <width>81</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Solape lateral</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="solapeLateralSpinBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="porcentaje2Label">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="espacioLineasLabel">
            <property name="text">
             <string>Espacio entre lineas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="espacioLineasTextEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>50</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="metros4Label">
            <property name="text">
             <string>metros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="distanciaTotalLabel">
            <property name="text">
             <string>Distancia total: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="distanciaRecorridoText">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="meters5Label">
            <property name="text">
             <string>metros</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="optimizarBoton">
            <property name="text">
             <string>Optimizar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Cámaras</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="cerrarBoton">
        <property name="text">
         <string>Cerrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aplicarBoton">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
